design/ceph/submariner-globalnet.md

```diff
@@ -0,0 +1,96 @@
+# Submariner Globalnet support in Rook
+
+
+## Use case
+As a rook-ceph user, I should be able to setup mirroing across multiple clusters with overlapping networks in order to safegurd my data in case of any disaster.
+
+
+## Background
+[Submariner](https://submariner.io/getting-started/) provides cross-cluster connectivity between two kubernetes clusters when they have different CIDRs. Many a times customers don't have control over the network configuration of the clusters, resulting in clusters with overlapping CIDRs (ServiceCIDR and ClusterCIDR). For such scenarios, Submariner provides another tool called [Globalnet](https://submariner.io/getting-started/architecture/globalnet/). It provides cross-cluster connectivity from pods to remote services using their global IPs.
+
+In order to support kubernetes clusters connected using Submariner-Globalnet, Rook needs to use the global IPs provided by Globalnet instead of the service cluster IP.
```

Comment:
```suggestion
In order to support Kubernetes clusters connected by an MCS API-compatible application, Rook needs to use "clusterset" IPs instead of the Services' cluster IPs.
```